src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -195,6 +195,7 @@ CreateCitusLocalTable(Oid relationId, bool cascadeViaForeignKeys, bool autoConve
 	 */
 	EnsureCoordinator();
 	EnsureTableOwner(relationId);
+	ErrorIfRelationIsAKnownShard(relationId);
```

Comment:
We actually don't need that here. We already check this for Citus Local Tables.
In the function `ErrorIfUnsupportedCreateCitusLocalTable`. 
Please see:
https://github.com/citusdata/citus/blob/master/src/backend/distributed/commands/citus_add_local_table_to_metadata.c#L486